start-from-zero/Javascript/src/views/common/common.js

```diff
@@ -0,0 +1,37 @@
+const FooterFixed = function(){
+    return `
+    <footer class='fixed-footer'>
+        <div class="container">
+            <p class="footer-paragraph">Copyright@softvision 2017</p>
+        </div>
+    </footer>
+    `;
+}
+
+const Footer = function(){
```

Comment:
there is a simple class toggle between these 2 components. Is it really worth duplicating the code for this?
What better options are there?